### PR TITLE
api: support delete member by id.

### PR DESCRIPTION
**Pull request by disksing**
related issue: https://github.com/pingcap/pd/issues/694

### CODE REVIEW

File: pdctl/command/member_command.go [hunk 1]
```diff
@@ -67,12 +75,26 @@ func showMemberCommandFunc(cmd *cobra.Command, args []string) {
...
+func deleteMemberByNameCommandFunc(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
 		fmt.Println("Usage: member delete <member_name>")
 		return
 	}
-	prefix := fmt.Sprintf(memberPrefix, args[0])
+	prefix := membersPrefix + "/" + args[0]
+	_, err := doRequest(cmd, prefix, http.MethodDelete)
+	if err != nil {
+		fmt.Printf("Failed to delete member %s: %s", args[0], err)
```
Comment (huachaohuang) [inline pdctl/command/member_command.go]: Should add "\n"?

File: server/api/router.go [hunk 2]
```diff
@@ -80,7 +80,9 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	router.Handle("/api/v1/status", newStatusHandler(rd)).Methods("GET")
 
 	router.Handle("/api/v1/members", newMemberListHandler(svr, rd)).Methods("GET")
-	router.Handle("/api/v1/members/{name}", newMemberDeleteHandler(svr, rd)).Methods("DELETE")
+	memberDeleteHandler := newMemberDeleteHandler(svr, rd)
+	router.HandleFunc("/api/v1/members/{name}", memberDeleteHandler.DeleteByName).Methods("DELETE")
```
Comment (huachaohuang) [inline server/api/router.go]: Is it better to use query string?
For example: `/api/v1/members?name={name}` or `/api/v1/members?id={id}`.

Comment (disksing) [inline server/api/router.go]: I think they are not much different.

Comment (overvenus) [inline server/api/router.go]: I agree with @huachaohuang, although that is incompatible with previous tools.

Comment (disksing) [inline server/api/router.go]: How about `/api/v1/members/name/{name}` and `/api/v1/members/id/{id}`?

Comment (disksing) [inline server/api/router.go]: cc @huachaohuang @overvenus

Comment (siddontang) [inline server/api/router.go]: `member_ids/{id}`

Comment (huachaohuang) [inline server/api/router.go]: Can't find a best practice after a simple search, but I tend to agree with `members/name/{name}` and `members/id/{id}` now.
Ref: https://stackoverflow.com/questions/842261/rest-supporting-multiple-possible-identifiers

**Comment by huachaohuang** [comment]
LGTM

**Pull request merged**